.gitignore

```diff
@@ -0,0 +1,13 @@
+# dependencies
+/node_modules
+
+# production
+/dist
+
+# misc
+.DS_Store
```

Comment:
Wish you could add `*.iml` and `.idea` :)

Reply:
Done!